packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -117,6 +132,17 @@ export default class QuanticRefineModalContent extends LightningElement {
     }
   }
 
+  /**
+   * Gather all sort options registered in the Quantic store.
+   * @returns {void}
+   */
+  gatherSortOptions() {
+    this.sortData = getAllSortOptionsFromStore(this.engineId);
+    if (this.sortData !== undefined && this.sortData.length > 0) {
```

Comment:
```suggestion
    if (this.sortData?.length) {
```

could be just this based on the truthy/falsy js logic.